chatbot_ner/config.py

```diff
@@ -35,6 +35,8 @@
                                                        'host': DESTINATION_HOST,
                                                        'port': DESTINATION_PORT})
 
+DISABLE_REPLICAS_WHILE_TRANFERRING = os.environ.get('DISABLE_REPLICAS_WHILE_TRANFERRING', True)
```

Comment:
Set the default value to False for consistency and the existing behavior will continue

Reply:
value set to False